package.json

```diff
@@ -16,7 +16,7 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "install": "node-gyp rebuild"
+    "install": "./scripts/install-execsync"
```

Comment:
Did you try using [optional dependencies](https://docs.npmjs.com/files/package.json#optionaldependencies)? I think that would be a cleaner way of optionally installing execSync.